command/common/internal/help_all_display.go

```diff
@@ -172,7 +173,7 @@ var ExperimentalHelpCategoryList = []HelpCategory{
 	{
 		CategoryName: "EXPERIMENTAL COMMANDS:",
 		CommandList: [][]string{
-			{"revisions", "revision", "rollback"},
```

Comment:
**issue** Revision should not be made unexperimental since it does not do anything

Reply:
Ah you're right, it was too aggressive to move _both_ of these commands out of the EXPERIMENTAL block. I also see now that [the ticket](https://jira.eng.vmware.com/browse/TPCF-10927) mentions in passing that the `rollback` command should also not be marked EXPERIMENTAL.

I'll submit another revision to all three of these PRs that:
- Restores `revision` to `EXPERIMENTAL` status.
- Moves `rollback` _out_ of `EXPERIMENTAL` status.

Reply:
Updated PR as promised:

- `cf revision` remains in `EXPERIMENTAL` status.
- `cf revisions` is no longer in `EXPERIMENTAL` status.
- `cf rollback` is no longer in `EXPERIMENTAL` status.